ocs_ci/deployment/multicluster_deployment.py

```diff
@@ -49,6 +51,17 @@ def do_deploy_ocp(self, log_cli_level="INFO"):
             return
         if self.multicluster_mode == constants.RDR_MODE:
             self.do_rdr_acm_ocp_deploy()
+        prev = config.cur_index
+        for cluster in get_non_acm_cluster_config():
+            config.switch_ctx(cluster.MULTICLUSTER["multicluster_index"])
+            ssl_key = config.DEPLOYMENT.get("ingress_ssl_key")
+            ssl_cert = config.DEPLOYMENT.get("ingress_ssl_cert")
+            for key in [ssl_key, ssl_cert]:
+                if os.path.exists(key):
+                    os.unlink(key)
+            logger.info("Running post ocp deploy ops")
+            self.post_ocp_deploy()
+        config.switch_ctx(prev)
```

Comment:
Can this be moved into a standalone function/method instead of being inside of `do_deploy_ocp` directly?

Reply:
Makes sense, I will make the change and rerun the verification